### PR TITLE
update to Sifts new Score response format

### PR DESCRIPTION
**Pull request by Jonahss**
Hello,
Sift updated the way format of the ScoreResponse.

### CODE REVIEW

**Comment by Jonahss** [comment]
@mathieukempe could you review and merge?

File: src/siftsciencenet.sln [hunk 1]
```diff
@@ -1,27 +1,33 @@
...
+VisualStudioVersion = 14.0.25420.1
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SiftScienceNet", "SiftScienceNet\SiftScienceNet.csproj", "{7D57EF7D-3AB6-458C-84C0-EB9867D58608}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = ".nuget", ".nuget", "{45BB0474-8EB4-43A0-8ECC-F5A38D46923E}"
+	ProjectSection(SolutionItems) = preProject
+		.nuget\packages.config = .nuget\packages.config
+	EndProjectSection
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SiftScienceNetTests1", "SiftScienceNetTests1\SiftScienceNetTests1.csproj", "{B5AD0DA4-51C6-4D07-9C70-DC6900E362EA}"
```
Comment (mathieukempe) [inline src/siftsciencenet.sln]: Just one thing, it fail to build because of a reference to the project SiftScienceNetTests1\SiftScienceNetTests1.csproj

**Comment by mathieukempe** [comment]
Thanks, I had a bit of time today to push the new version to nuget

**Comment by Jonahss** [comment]
Oh thanks! sorry I've been pretty busy too.

**Pull request merged**